README.md

```diff
@@ -719,8 +719,10 @@ The web service features is disabled by default and must be explicitly activated
 
 ### Factory reset
 
+Description [Factory reset](#factory-reset).
+
 ```
-curl -X POST --unix-socket /run/omnect-device-service/api.sock http://localhost/factory-reset/v1
+curl -X POST --unix-socket /run/omnect-device-service/api.sock http://localhost/factory-reset/v1 -d '{"mode": 1, "preserve": []}'
```

Comment:
i prefer `--json` instead of `-d` to make it more obvious 
(it also sets the content type, which we currently seem to ignore then.)